### DISCUSSION TITLE
[manager/nuget] packages.lock file should be updated when dotnet-sdk is updated

### DISCUSSION THREAD

**Discussion by marcmognol** [renovatebot/renovate#27977] [Suggest an Idea]
### Tell us more.

When a new version of dotnet sdk is released, Renovate creates a new PR to update the global.json.
But Renovate doesn't update the packages.lock.json file, and as some nuget packages are updated along the sdk, our CI pipelines fails and I need to update packages.lock json file manually.

I would like Renovate run a `dotnet restore` whenever the global.json is updated.

Thanks

**Comment by rarkins** [comment]
@viceice wdyt?

**Reply by viceice** [reply]
sounds plausible 👍, we can probably simply make all project files depending on the global json, like we do for central versioning. that way renovate knows what projects need a restore call

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/28006
